client/signup/steps/store-features/intents.tsx

```diff
@@ -9,18 +8,14 @@ import { StoreFeatureSet } from './types';
 
 export function useIntents(
 	siteSlug: string | null,
-	sitePlanSlug: string | undefined,
+	hasPaymentsFeature: bool | false,
```

Comment:
I think `bool` needs to be `boolean` to avoid throwing type errors.

Reply:
Yeah! Thanks :)